sso/sso.js

```diff
@@ -0,0 +1,109 @@
+const router = require('express').Router()
+const config = require('../util/config')
+const { auth, registry } = require('../resources/nuts-node')
+
+const patientResource = require('../resources/database').patient
```

Comment:
Misschien fijn om hier gewoon consequent te zijn met de rest van het project

Reply:
Die "patient" is geen patient maar een patientStore / patientRepository / patientResource. Vervolgens haal ik een daadwerkelijk patient op uit de db, hoe moet ik die dan noemen?

Reply:
Ik bedoelde vooral qua syntax, niet zozeer qua naamgeving. Maar ik snap dat dat samenhangt in dit geval. Ik heb dat doorgaans opgelost (meen ik) door `patient` te gebruiken voor de repository en `patientObject` ofzo voor de patient instantie.